### PR TITLE
Created "section heading" styles that are used throughout

### PR DESCRIPTION
**Pull request by thisIvanRepo**

### CODE REVIEW

File: src/css/base/base.css [hunk 1]
```diff
@@ -36,6 +36,46 @@ main:has(.back-link) {
   padding: 0 20px;
 }
 
+.section__head {
+  /* margin-top: 80px; */
```
Comment (thisIvanRepo) [inline src/css/base/base.css]: I think this property is used in "section" styles.

**Comment by ElephantFrontend** [comment]
Waiting on changes

File: src/css/base/base.css [hunk 2]
```diff
@@ -36,6 +36,46 @@ main:has(.back-link) {
   padding: 0 20px;
 }
```
Comment (ElephantFrontend) [inline src/css/base/base.css:37]: waiting on changes

Comment (ElephantFrontend) [reply src/css/base/base.css:37]: fixed

**Pull request merged**